eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/ManualReferenceTestRunner.java

```diff
@@ -39,7 +39,7 @@
    * <p>e.g. set to "ssz_static" to run only ssz static tests or "ssz_static/Attestation" for only
    * attestation ssz tests.
    */
-  private static final String TEST_TYPE = "operations/execution_payload";
+  private static final String TEST_TYPE = "ssz_generic/basic_vector";
```

Comment:
maybe you do not want to hardcode this

Reply:
This is just our internal "manual runner" we use to run specific tests on demand while we are implementing them. It doesn't effect anything about the actual reference test run in CI.